packages/block-library/src/social-link/edit.js

```diff
@@ -88,6 +88,13 @@ const SocialLinkEdit = ( {
 		},
 	} );
 
+	const onSetLinkRel = useCallback(
+		( value ) => {
+			setAttributes( { rel: value } );
+		},
+		[ setAttributes ]
+	);
```

Comment:
Suggestion (Non-blocking): This is probably unnecessary optimization. We're not passing callback deep down the tree, and `TextControl` will re-render anytime `SocialLinkEdit` re-renders.

Reply:
Good catch, cleaned that up.